corehq/apps/app_manager/models.py

```diff
@@ -4859,6 +4860,13 @@ def wrap(cls, data):
                            if module.get('doc_type') != 'CareplanModule']
         self = super(Application, cls).wrap(data)
 
+        translations = data.get('translations')
+        label_dict = {lang: label.get('case.search.title')
+            for lang, label in translations.items() if label}
+        for module in self.modules:
+            if hasattr(module, 'search_config'):
+                setattr(getattr(module, 'search_config', 'title label', label_dict))
```

Comment:
```suggestion
                setattr(getattr(module, 'search_config', 'title_label', label_dict))
```
also check out the test failures